eggs1.rst

```diff
@@ -184,7 +176,7 @@ Before we can use our new package we have to tell Plone about it. Edit ``buildou
     test-eggs +=
         ploneconf.site [test]
 
-This tells Buildout to add the egg ``ploneconf.site``. Since it is also in the `sources` section buildout will not try to download it from pypi but will expect it in ``src/ploneconf.site``. *fs* allows you to add packages on the filesystem without a version control system, or with an unsupported one.
+This tells Buildout to add the egg ``ploneconf.site``. Since it is also in the `sources` section buildout will not try to download it from pypi but will expect it in ``src/ploneconf.site``. *fs* allows you to add eggs on the filesystem without a version control system, or with an unsupported one.
```

Comment:
An egg seems to be a package with metadata and since we're talking buildout I think we'd off with the term it adopts for its keys: `eggs` and `develop-eggs`, among others.